### DISCUSSION TITLE
Problems in the way Data and GIS is Used

### DISCUSSION THREAD

**Discussion by nnhimichelle** [albertkun/21S-ASIAAM-191A#28] [Past Thinking Caps]
Data and GIS are easily misinterpreted depending on the institution, organization, or individual that use it. They can easily make assumptions and generalizations from the data that can create conflict between communities or create harm within them. For instance, data with Asian American and Pacific Islander communities are incredibly difficult to disaggregate, so clumping together those communities into one disregards the disparities that South, East, Southeast, and other Asian communities face. This creates such skewed data for the demographic and perpetuates the Model Minority Myth already used to form divisions between marginalized communities. Misinformation and false narratives can easily be judged from looking at the maps, assuming that because it’s all in the picture that it could speak the thousand words everyone else sees too.

**Comment by ghost** [comment]
I think the points you made above are extremely important. Demographic-based data is probably some of the most disproportionately skewed or misrepresented information. When using this data to create maps there are myths, stereotypes, and more that get perpetuated, like in the instance you mentioned. While being an amazing visualization and analysis tool, GIS can also be used negatively whether it be through conscious or unconscious data portrayal. I think it is important that you specifically pointed out how this affects Asian and Pacific Islander communities because there is a lot of misconception surrounding these groups which are only furthered by maps and data that are made to look convincing of distribution or traits that likely aren't being represented correctly.

**Comment by KenNL42** [comment]
I agree that data tends to be generalized and lot of useful information can be lost. From your example, I also agree that although Asian and Pacific Islanders may cover the whole group, but there are lots of difference within the group that can easily be classified further. This also leads me to think how other data can be generalized while losing some information. For example, the data of income gap may indicate how much top 1% make more than the rest, but we still do not know the rest of demographic. The density of income graph could be either left-skewed, right-skewed, or normal. The audience then do not see the full picture of what the data says about income gap.
